src/pages/components/progress-bar/usage.mdx

```diff
@@ -0,0 +1,504 @@
+
+As a more concrete set of examples, there are generally three different major
+progressions of a system operation: downloading, uploading, and local
+processing.
+
+#### Downloading
+
+For downloading actions it's usually possible to get a value of progress and
+update it in real time for a determinate progress bar. Depending on the
+information reported from a data source, time remaining could be estimated based
+on how much has been downloaded so far over a time period.
+
+#### Uploading
+
+Upload actions may be unable to report real time progress status depending on
+the data source and API configuration. However, always strive to provide users
+the determinate value whenever it's possible to inform them about the process.
+
+#### Data processing
+
+Data processing refers to manipulations of data to produce meaningful
+information. This can include currency exchange, file generation, data
+reformatting, or similar operations. Depending on the operation, time estimated
+or cycles to complete processing could be used to populate a determinate
+progress bar.
+
+## Formatting
+
+Determinate and indeterminate progress bars both have a label and an optional
+helper text. However, the inderterminate can’t provide the exact, quantiative
+information about the progression in the optional helper text.
+
+### Anatomy
+
+<Row>
+<Column colLg={8}>
+
+![Progress bar anatomy](images/progress-bar-usage-6.png)
+
+</Column>
+</Row>
+
+1. **Label:** The text describes the process that the progress bar refers to.
+   Label can be visually hidden, but requires to be defined to be accessible.
+2. **Track:** The static area that the bar indicator moves on top of and acts as
+   a fixed visual reference of what the total length and duration of the process
+   could be.
+3. **Bar indicator:** Indicates how much the process has progressed.
+4. **Helper text (optional):** Assistive text to show additional information
+   about the process taking place. For determinate, the information is usually
+   values of number or percentage. Sometimes, a fraction or ratio may be shown.
+   These values are not applicable when a progress bar is indeterminate.
+
+### Sizing
+
+#### Height
+
+The progress bar is offered in two different sizes—default (8px) and small
+(4px). The default progress bar height is typically used when there is large
+amounts of space on a page. The small progress bar height is commonly used when
+space is restricted and can be placed within cards, data table, or side panels.
+
+<Row>
+<Column colLg={8}>
+
+![Progress bar sizes](images/progress-bar-usage-7.png)
+
+</Column>
+</Row>
+
+#### Width
+
+The width of a progress bar can be customized appropriately for its context. The
+minimum width of a progress bar is 48px and keep its width to a maximum of six
+columns when possible. If the progress bar width is too long it can reduce
+readability. Don't increase the progress bar length to fill the entire width of
+a window or application.
```

Comment:
Brief suggestion

```suggestion
The width of a progress bar can be customized appropriately for its context. The
minimum width of a progress bar is 48px and it should be kept to a maximum of six
columns wide when possible. If the progress bar width is too long it can reduce
readability. Don't increase the progress bar length to fill the entire width of
a window or application.
```